src/controllers/update/remove.cannot.use.controller.ts

```diff
@@ -7,15 +7,21 @@ export const get = (req: Request, res: Response, next: NextFunction) => {
     logger.debugRequest(req, `GET ${config.SIGN_OUT_PAGE}`);
 
     let backLinkUrl = "";
+    let signOutExtraQueryParams = "";
+
     const previousPage = req.query[config.PREVIOUS_PAGE_QUERY_PARAM];
     if (previousPage) {
       backLinkUrl = `${previousPage}${config.JOURNEY_REMOVE_QUERY_PARAM}`;
+      signOutExtraQueryParams = `${config.PREVIOUS_PAGE_QUERY_PARAM}=${previousPage}`;
     }
 
     return res.render(config.REMOVE_CANNOT_USE_PAGE, {
       journey: config.JourneyType.remove,
       backLinkUrl,
-      templateName: config.REMOVE_CANNOT_USE_PAGE
+      templateName: config.REMOVE_CANNOT_USE_PAGE,
+      url: `${config.UPDATE_AN_OVERSEAS_ENTITY_URL}`, // used in sign-out-user-banner to build correct url for sign out page
```

Comment:
Do you need to do this? It looks as if the template logic will set it anyway if missing.